src/gluonts/dataset/repository/datasets.py

```diff
@@ -23,6 +23,7 @@
     generate_gp_copula_dataset,
 )
 from gluonts.dataset.repository._lstnet import generate_lstnet_dataset
+from gluonts.dataset.repository._zenodo import generate_forecasting_dataset
```

Comment:
`zenodo` is probably a bad name. If the data was on drop-box, would you name this `dropbox`?

Reply:
Yes, you are right, changed to `_tsf_datasets`